packages/manager/modules/pci/src/projects/project/storages/cold-archive/archives/onboarding/onboarding.controller.js

```diff
@@ -0,0 +1,46 @@
+import reduce from 'lodash/reduce';
+import illustration from './assets/ColdStorage.png';
+import { COLD_ARCHIVE_TRACKING_PREFIX } from '../../cold-archives.constants';
+import { GUIDES } from './onboarding.constants';
+
+export default class PciStorageObjectsOnboardingController {
+  /* @ngInject */
+  constructor($translate, atInternet) {
+    this.$translate = $translate;
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.illustration = illustration;
+    this.guides = reduce(
+      GUIDES,
+      (list, guide) => [
+        ...list,
+        {
+          ...guide,
+          title: this.$translate.instant(
+            `pci_projects_project_storages_cold_archives_onboarding_guides_${guide.id}_title`,
+          ),
+          description: this.$translate.instant(
+            `pci_projects_project_storages_cold_archives_onboarding_guides_${guide.id}_description`,
+          ),
+        },
+      ],
+      [],
+    );
+  }
+
+  addColdArchive() {
+    this.atInternet.trackClick({
```

Comment:
Suggestion: if we are going to use tracking in future at other places as well, then better we can put it at a common place and consume it from there, like in cold-archive routing.